### ISSUE TITLE
About clip_norm 

### ISSUE THREAD

**Issue by LMMMEng** [sail-sg/poolformer#47]
Thanks for the excellent work and Happy Chinese New Year!

I noticed that you didn't use the clip_norm that is commonly used in your previous works. Why?

**Comment by yuweihao** [comment]
Hi @LMMMEng ,

For PoolFormer, Gradient Clipping (optimizer: AdamW) cannot see significant improvement. Following the hyper-parameters of DeiT, we also do not use Gradient Clipping.

**Comment by LMMMEng** [comment]
Thank you for your reply.
